src/components/UseWhenYouNeedThis.css

```
@@ -0,0 +1,39 @@
+.use-when-list {
+  list-style: none;
+  margin: 0 0 1.5rem;
+  padding: 0;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  gap: 0.75rem;
+}
+
+.use-when-list li {
+  padding: 0.75rem 1rem;
+  background: rgba(158, 163, 175, 0.08);
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)